index.html

```diff
@@ -12,32 +12,165 @@
 <body>
     <header class="header">
         <nav class="nav">
-            <a href="/" class="john-doe">John Doe</a>
+            <a href="/" class="john-doe">Ademibowale</a>
             <nav class="header-bar"><i class=" fa fa-bars cursor "></i></nav>
         </nav>
     </header>
     <section class="main-section ">
         <div>
```

Comment:
- [ ] You should add your section tags inside main tag as their parent tag.  This will improve you HTML semantics, and to read more about it you can read [here](https://www.w3schools.com/tags/tag_main.asp#:~:text=Definition%20and%20Usage,site%20logos%2C%20and%20search%20forms.)

You can use this format : 
`<header></header`
`<main></main>`
`<footer></footer>`